### PR TITLE
chore: add to git blame ignore

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

prevent git-submodule to subtree migration commit from showing up in git blame history

- take out an unnecessary use of getPath

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

### CODE REVIEW

File: webui/react/src/pages/SignIn.tsx [hunk 1]
```diff
@@ -72,9 +71,8 @@ const SignIn: React.FC = () => {
       if (queries.cli) notification.open({ description: <AuthToken />, duration: 0, message: '' });
 
       // Reroute the authenticated user to the app.
-      const loginRedirect = getPath<Location>(location, 'state.loginRedirect');
       if (!queries.redirect) {
-        routeToReactUrl(locationToPath(loginRedirect) || defaultRoute.path);
+        routeToReactUrl(locationToPath(location.state?.loginRedirect) || defaultRoute.path);
```
Comment (keita-determined) [inline webui/react/src/pages/SignIn.tsx:75]: should be `??` instead of `||`?

Comment (keita-determined) [reply webui/react/src/pages/SignIn.tsx:75]: nvm 
i didnt see its in `locationToPath` func 😰

**Comment by trentwatt** [comment]
am i doing it wrong?

https://user-images.githubusercontent.com/28383080/195835874-fe978353-bd1d-4985-8233-1e7e18426155.mov

**Comment by keita-determined** [comment]
@trentwatt 
are you trying to ignore the commit? 
If so, ` git blame --ignore-revs-file .git-blame-ignore-revs <file_name>`

File: webui/react/src/pages/SignIn.tsx [hunk 2]
```diff
@@ -18,7 +18,6 @@ import LogoOkta from 'shared/assets/images/logo-sso-okta-white.svg';
 import { StoreActionUI } from 'shared/contexts/UIStore';
```
Comment (ClaireNeveu) [inline webui/react/src/pages/SignIn.tsx:18]: These changes seem unrelated?

Comment (hamidzr) [reply webui/react/src/pages/SignIn.tsx:18]: true. it's related to what I was chatting with Keita about but not this. just being lazy about making two PRs for something that __should__ essentially be a no-op. it would be safer to have two separte ones..

Comment (hamidzr) [reply webui/react/src/pages/SignIn.tsx]: I went ahead and split that into its own pr
